### ISSUE TITLE
--kmer_count=1 causes division by zero

### ISSUE THREAD

**Issue by sjackman** [rrwick/Unicycler#94]
```
❯❯❯ unicycler --kmer_count=1 --min_kmer_frac=0.68 --max_kmer_frac=0.68 …
…
Choosing k-mer range for assembly (2018-03-07 17:11:59)
    Unicycler chooses a k-mer range for SPAdes based on the length of the input
reads. It uses a wide range of many k-mer sizes to maximise the chance of
finding an ideal assembly.

Traceback (most recent call last):
  File "/gsc/btl/linuxbrew/bin/unicycler", line 11, in <module>
    load_entry_point('unicycler==0.4.3', 'console_scripts', 'unicycler')()
  File "/gsc/btl/linuxbrew/opt/python3/lib/python3.6/site-packages/unicycler/unicycler.py", line 87, in main
    args.no_correct, args.linear_seqs)
  File "/gsc/btl/linuxbrew/opt/python3/lib/python3.6/site-packages/unicycler/spades_func.py", line 73, in get_best_spades_graph
    max_k_frac)
  File "/gsc/btl/linuxbrew/opt/python3/lib/python3.6/site-packages/unicycler/spades_func.py", line 458, in get_kmer_range
    for x in [x / (kmer_count - 1) for x in range(kmer_count)]:
  File "/gsc/btl/linuxbrew/opt/python3/lib/python3.6/site-packages/unicycler/spades_func.py", line 458, in <listcomp>
    for x in [x / (kmer_count - 1) for x in range(kmer_count)]:
ZeroDivisionError: division by zero
```
For context see https://github.com/rrwick/Unicycler/issues/92#issuecomment-371340333

**Comment by rrwick** [comment]
Good catch! Fixed in e609656 (currently just development branch, but should make it into the next release).

Ryan

**Comment by sjackman** [comment]
Thanks, Ryan!
